lib/gate/rails.rb

```diff
@@ -22,7 +27,7 @@ def fetch_params_schema(schema_name)
       end
 
       def def_schema(&block)
-        @_schema = Dry::Validation.Params(&block)
+        @_schema = Dry::Validation.Params(BaseSchema, &block)
```

Comment:
Whoa, didn't know that. Is it documented? :D

Reply:
Well, documentation don't know everything :D 
https://github.com/dry-rb/dry-validation/blob/master/lib/dry/validation.rb#L27